src/_index.scss

```
@@ -159,8 +159,13 @@ $warning_color: $BANANA_900;
         $text-size: $text-size * 1px;
     }
 
-    // Workaround GTK clamping instead of rounding up
-    @return ($pixels / $text-size * 1rem) + 0.000000001rem;
+    @if ($pixels > 1) {
```

If this is supposed to be a switch for negative numbers, then this makes more sense… right?

```suggestion
    @if ($pixels > 0) {
```